### PR TITLE
49 add functionality to menu left

### PR DESCRIPTION
**Pull request by Ewar22**
# Description
>Add functionality to menu left
-add functionality to the category search via the menu
## Screenshots
image
image

### CODE REVIEW

**Review by claunicole — APPROVED** [review]
good work

**Review by EverardoAlcarazBaltazar — APPROVED** [review]

File: app/assets/stylesheets/pages/_main.scss [hunk 1]
```diff
@@ -77,6 +78,27 @@
...
+          background: $color1;
+          border: 0px;
+          width: 20%;
+        }
+        
+        .logo__search {
+          width: 50%;
+          height: auto;
+          background: $color1;
+          margin-top: 1rem;
```
Comment (oscaregr) [inline app/assets/stylesheets/pages/_main.scss:100]: sort this.

File: app/controllers/mainpages_controller.rb [hunk 2]
```diff
@@ -26,5 +35,10 @@ def user_following
       follower = current_user.follower&.split(",")&.map(&:to_i)
       User.all.where(id: follower)
     end
+        
+    # def category
+    #     param = params[:category_id].nil? ? 1 : params[:category_id]
+    #     Category.find_by(id: param.to_i)
+    # end
```
Comment (oscaregr) [inline app/controllers/mainpages_controller.rb:42]: Delete this comment

**Review by oscaregr — APPROVED** [review]
Resolve the comments

**Pull request merged**